### ISSUE TITLE
Bounty currency symbol?

### ISSUE THREAD

**Issue by Procrat** [ZeusWPI/gamification#67]
We could use the [Thai Baht](https://en.wikipedia.org/wiki/Thai_baht) symbol instead of "bounty points", for both brevity and elimination of any confusion with bounty score.

Example usage:
> @Iasoon placed ฿50 on gamifcation#67: Bounty currency symbol?

Any thoughts?

**Comment by iasoon** [comment]
I like this!

**Comment by iasoon** [comment]
However, isn't that symbol often (erroneously) used to represent BitCoin? In that case, we could consider using another one to not cause any confusion.

**Comment by Procrat** [comment]
Yeah, it happens. I think the confusion will quickly subside once they want to know more about what those message on Slack mean.

I just discussed this with Myrjam and she raises the relevant question whether newbies will actually click on these mentions or just ignore them. The reason for ignoring might be that they don't know what bounties are (at least nog in this context), what GitHub is, what GitHub issues are and what gamification in general does (at least nog from that sole message on Slack). Posting "฿50" instead of "bounty points" makes it perhaps even more obscure.
Do you think this is an issue? Could we overcome this?

**Comment by iasoon** [comment]
When you'd be willing to work on Zeus projects, I guess you will easily find out. This would be an issue when our users existed in a vacuum, but fortunately we see them every few days, and even engage in conversation with them.
In short, I do not think that sole message on Slack *should* explain everything.

**Comment by Procrat** [comment]
> When you'd be willing to work on Zeus projects
We're at the point of trying to encourage them of doing so.

**Comment by feliciaan** [comment]
@Procrat but bounty points are only shown in the #zeus-dev channel, so these notifications would only be for people who already joined the #zeus-dev channel. And these people would have already some experience with Zeus projects.

**Comment by iasoon** [comment]
@Procrat I doubt anyone would start contributing because of gamification. I expect it to encourage more contributions, but there has to be some engagement to start with.

**Comment by Procrat** [comment]
I guess you're right. It somehow feels a bit like a failure though.

**Comment by iasoon** [comment]
That is, some engagement with Zeus, not neccisarily with our projects. In that case, you'd still know what Gamification is.

**Comment by iasoon** [comment]
Bumping this in relation to #87.

**Comment by NoctuaNivalis** [comment]
How about:
- ₱ for Points. [Philippine peso](https://en.wikipedia.org/wiki/Philippine_peso)
- ¤, the generic currency sign.

**Comment by Silox** [comment]
Or a combination? ฿₱ for ฿ounty₱oints?
